tests/MyGame/MonsterExtra.py

```diff
@@ -60,11 +60,23 @@ def TestdNinf(self):
             return self._tab.Get(flatbuffers.number_types.Float64Flags, o + self._tab.Pos)
         return float('-inf')
 
-def MonsterExtraStart(builder): builder.StartObject(6)
+    # MonsterExtra
+    def Testa(self):
+        o = flatbuffers.number_types.UOffsetTFlags.py_type(self._tab.Offset(16))
+        if o != 0:
+            x = o + self._tab.Pos
+            from .ArrayStruct import ArrayStruct
```

Comment:
why is this import in the middle of the code? is that a normal thing to do in Python? :)

Reply:
Again this is seems to be generated when a struct is added to a table. It was not introduced with this patch set :)